src/agentscope/manager/_file.py

```diff
@@ -8,11 +8,11 @@
 import numpy as np
 from PIL import Image
 
-from agentscope.utils.tools import _download_file
-from agentscope.utils.tools import _hash_string
-from agentscope.utils.tools import _get_timestamp
-from agentscope.utils.tools import _generate_random_code
-from agentscope.constants import (
+from ..utils.common import _download_file
+from ..utils.common import _hash_string
+from ..utils.common import _get_timestamp
+from ..utils.common import _generate_random_code
```

Comment:
How about import multi-object just like the line below?

Reply:
DONE